spec/rubocop/cop/rspec/factory_bot/create_list_spec.rb

```diff
@@ -189,6 +189,48 @@
         create_list(:user, 3) { |user| create(:account, user: user) }
       RUBY
     end
+
+    context 'with empty array' do
+      it 'does not register an offense' do
+        expect_no_offenses(<<~RUBY)
+          []
+        RUBY
+      end
+    end
+
+    context 'with different `create` nodes in array' do
+      it 'does not register an offense' do
+        expect_no_offenses(<<~RUBY)
+          [create(:user), create(:user, age: 18)]
+        RUBY
+      end
+    end
+
+    context 'with one `create` node in array' do
+      it 'registers and corrects an offense' do
+        expect_offense(<<~RUBY)
+          [create(:user)]
+          ^^^^^^^^^^^^^^^ Prefer create_list.
+        RUBY
+
+        expect_correction(<<~RUBY)
+          create_list(:user, 1)
+        RUBY
+      end
+    end
+
+    context 'with same `create` nodes in array' do
+      it 'registers and corrects an offense' do
+        expect_offense(<<~RUBY)
+          [create(:user), create(:user)]
```

Comment:
I'm curious how the cop would behave with more complicated cases, e.g.

```ruby
[create(:user, age: 18), create(:user, age: 18)]
```
and especially with
```ruby
[create(:user, age: rand(18..32)), create(:user, age: rand(18..32))]
```

Reply:
Good point. I'll add this kind of test cases. Actually, this will always be replaced with n_times style, and it will not be replaced with create_list style because it contains a method call.

```ruby
2.times { create(:user, age: rand(18..32) }
```

The following points must be considered in this case:

1. Should it really be replaced when `EnforcedStyle: create_list`?
    - I believe it would be good to replace in this case. This is fine as currently implemented.
2. What message should be displayed in the offense?
    - The current implementation simply shows a message corresponding to the EnforcedStyle, which is at odds with the final autocorrection result, so it would be better to calculate whether the method call is included inside and display the appropriate message.